Epsilon.Canvas/Rest/PageEndpoint.cs

```diff
@@ -1,16 +1,30 @@
 using System.Net;
 using System.Net.Http.Json;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 using Epsilon.Canvas.Abstractions.Rest;
 
 namespace Epsilon.Canvas.Rest;
 
 public class PageEndpoint : IPageEndpoint
 {
     private readonly HttpClient _client;
+    private readonly JsonSerializerOptions _serializerOptions;
 
     public PageEndpoint(HttpClient client)
     {
         _client = client;
+        _serializerOptions = new JsonSerializerOptions { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault, };
```

Comment:
We can make this` private static readonly` to make sure that it is not constantly reconstructed.